### PR TITLE
OPENEUROPA-2667: Create bundles for custom content entities.

### PR DESCRIPTION
**Pull request by nagyad**
**Attention**: https://github.com/openeuropa/oe_content/pull/147 to be merged at first.
Created the following bundles: 
- Contact entity
  - General
  - Press
- Organisation entity
  - Partner
- Venue entity
  - Default

### CODE REVIEW

File: modules/oe_content_entity/modules/oe_content_entity_contact/config/install/core.entity_form_display.oe_contact.general.default.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - field.field.oe_contact.general.oe_contact_phone
+    - oe_content_entity_contact.oe_contact_type.general
+  module:
+    - address
+id: oe_contact.general.default
+targetEntityType: oe_contact
+bundle: general
+mode: default
+content:
+  oe_contact_address:
```
Comment (ademarco) [inline modules/oe_content_entity/modules/oe_content_entity_contact/config/install/core.entity_form_display.oe_contact.general.default.yml]: I'm having some trouble with naming fields, I'll explain: `oe_contact` is the machine name of the entity type, prefixing each field with that would be like prefixing each node field with `node`. Let's use, as a general rule, that we don't prepend field names with entity type IDs.

Still, since these are the fields we ship with, let's use the same convention we used up until now and prepend them with `oe_`.

To summarize let's apply the following changes to all field names, on all entity types:

- `oe_contact_address` -> `oe_address` 
- `oe_contact_email` -> `oe_email` 
- `oe_contact_phone` -> `oe_phone` 
- etc.

File: modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/core.entity_form_display.oe_organisation.partner.default.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  oe_organisation_partner_image:
+    weight: 2
+    settings:
+      match_operator: CONTAINS
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: entity_reference_autocomplete
+    region: content
+  oe_organisation_partner_web:
```
Comment (ademarco) [inline modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/core.entity_form_display.oe_organisation.partner.default.yml]: Let's use `oe_link` as machine name, label stays the same.

File: modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/core.entity_form_display.oe_organisation.partner.default.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - field.field.oe_organisation.partner.oe_organisation_partner_web
+    - oe_content_entity_organisation.oe_organisation_type.partner
+  module:
+    - link
+id: oe_organisation.partner.default
+targetEntityType: oe_organisation
+bundle: partner
+mode: default
+content:
+  oe_organisation_partner_image:
```
Comment (ademarco) [inline modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/core.entity_form_display.oe_organisation.partner.default.yml]: Let's use `oe_image` as machine name, label stays the same.

File: modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/field.field.oe_organisation.oe_partner.oe_web.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.oe_organisation.oe_web
+    - oe_content_entity_organisation.oe_organisation_type.oe_partner
+  module:
+    - link
+id: oe_organisation.oe_partner.oe_web
```
Comment (ademarco) [inline modules/oe_content_entity/modules/oe_content_entity_organisation/config/install/field.field.oe_organisation.oe_partner.oe_web.yml]: Let's rename this into `oe_link`

**Comment by ademarco** [comment]
README.md is missing, please read my last review comments.
